src/main/java/core/basesyntax/CustomMathPow.java

```diff
@@ -9,6 +9,17 @@
      *      exponential - значение степени
      */
     public int pow(int number, int exponential) {
-        return 0;
+        int count = 1;
+        int result = number;
+
+        for (int i = 1; i < exponential; i++) {
+            if (exponential == 0) {
```

Comment:
@4MoPDai Maybe it will be better to move this if out from the loop? If exponential == 0 how you will go over the loop?